### DISCUSSION TITLE
how to fix faces, hands in Fooocus

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#3368] [Q&A]
a naturally lighted classroom with with high school kids doing Yoga looking out towards a garden. Partial wood ceiling. Warm color palette. A mix of linear aner and round pendant lights. Wood floor with area rugs.

![2024-07-26_15-40-30_6138](https://github.com/user-attachments/assets/52f56568-5fda-4619-9ad3-5ad810590edb)
![2024-07-26_15-40-14_9635](https://github.com/user-attachments/assets/8b33501f-46c4-4696-b51a-54af08ef285a)

I think we can improve it with inpaint, or so but not sure and how to implement it in Fooocus .

Appreciate your help.

**Comment by mashb1t** [comment]
Try to use Enhance twice, first arms then hands.

**Reply by oldhand7** [reply]
Hi, mashb1t, could you tell me more how to do?
Perhaps it's style or LoRA?

**Comment by lgwjames** [comment]
i also use the lora "perfect hands" and "perfect eyes"
